wrappers/python/tests/TestOpcWater.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import unittest
+import openmm as mm
+from openmm import app
+from openmm import unit
+
+
+class TestOpcWater(unittest.TestCase):
```

Comment:
Rather than creating a separate class for testing these two water models, let's put the tests in TestForceField.py.  It already contains tests for specific force field XML files.

Reply:
@peastman Okay, I moved them over